Packs/MicrosoftGraphIdentityandAccess/Integrations/MicrosoftGraphIdentityandAccess/MicrosoftGraphIdentityandAccess.py

```diff
@@ -255,6 +330,162 @@ def remove_member_from_role(client: Client, args: dict) -> str:
+            'displayName': display_name,
+            'isTrusted': is_trusted,
+            'ipRanges': ips_arr
+        }
+        if results := ms_client.create_ip_named_location(data):
+            id = results.get('id')
+            context = {
+                'id': id,
+                'display_name': results.get('displayName'),
+                'time_created': results.get('createdDateTime'),
+                'time_modified': results.get('modifiedDateTime'),
+                'is_trusted': results.get('isTrusted'),
+                'ip_ranges': results.get('ipRanges')
+            }
+            return CommandResults(
+                'MSGraph.conditionalAccess.namedIpLocations',
+                'namedIpLocations',
+                outputs=context,
+                raw_response=results,
+                readable_output=tableToMarkdown(
+                    f'created Ip named location \'{id}\':',
+                    context
+                )
+            )
+    return CommandResults(readable_output="Could not create ip named location")
+
+
+def ip_named_location_delete(ms_client: Client, args: dict) -> CommandResults:
+    if ip_id := args.get('ip_id'):
+        if results := ms_client.delete_ip_named_location(ip_id): # noqa
+            return CommandResults(
+                'MSGraph.conditionalAccess.namedIpLocations',
+                'namedIpLocations',
+                outputs={},
+                raw_response={},
+                readable_output=tableToMarkdown(
+                    f'Trying to delete ip named location \'{ip_id}\':', {'Success': 'True'})
+            )
+    return CommandResults(readable_output=tableToMarkdown(
+        f'Trying to delete ip named location \'{ip_id}\':', {'Success': 'False'}))
+
+
+def ip_named_location_list(ms_client: Client, args: dict) -> CommandResults:
+    limit = args.get('limit', '100')
+    if results := ms_client.list_ip_named_location(limit):
+        ip_named_locations = []
+        for result in results:
+            ip_named_location = {
+                'id': result['id'],
+                'display_name': result['displayName'],
+                'time_created': result['createdDateTime'],
+                'time_modified': result['modifiedDateTime'],
+                'is_trusted': result['isTrusted'],
+                'ip_ranges': result['ipRanges']
+            }
+            ip_named_locations.append(ip_named_location)
+        context = {
+            'ip_named_locations': ip_named_locations
+        }
+        return CommandResults(
+            'MSGraph.conditionalAccess.namedIpLocations',
+            'namedIpLocations',
+            outputs=context,
+            raw_response=results,
+            readable_output=tableToMarkdown(
+                'Ip named locations:',
+                ip_named_locations,
+            )
+        )
+    else:
+        return CommandResults(readable_output="could not list ip named locations")
+
+
+def ms_ip_string_to_list(ips):
```

Comment:
add type annotation (why do you need to cast it to str?)

Reply:
Idid the casting so I won't get an error when I do the split @yaakovi

Reply:
Why would you get an error if the variable is already a string?
According to the type annotation, it is always expected to be a string.